app/src/ui/tutorial-panel/tutorial-panel.tsx

```diff
@@ -0,0 +1,165 @@
+            onClick={this.handleToggle}
+          >
+            <div>
+              It doesn’t look like you have a text editor installed. We can
+              recommend{' '}
+              <LinkButton uri="https://atom.io" title="Open the Atom website">
+                Atom
+              </LinkButton>
+              {` or `}
+              <LinkButton
+                uri="https://code.visualstudio.com"
+                title="Open the VS Code website"
+              >
+                Visual Studio Code
+              </LinkButton>
+              , but feel free to use any.
+            </div>
+            <LinkButton>I have an editor</LinkButton>
+          </ListItem>
+          <ListItem
+            summaryText="Make a branch"
+            id="step-2"
+            openId={this.state.openId}
+            onClick={this.handleToggle}
+          >
+            <div>
+              Create a branch by going into the branch menu in the top bar and
+              clicking New Branch.
+            </div>
+            <kbd>⇧⌘N</kbd>
+          </ListItem>
+          <ListItem
+            summaryText="Edit a file"
+            id="step-3"
+            openId={this.state.openId}
+            onClick={this.handleToggle}
+          >
+            <div>
+              Open this repository in your preferred text editor. Edit the{' '}
+              <Monospaced>README.md</Monospaced> file, save it, and come back.
+            </div>
+            <Button
+              onClick={this.openFileInEditor}
+              disabled={!this.props.externalEditorLabel}
+            >
+              Open Editor
+            </Button>
+            <kbd>⇧⌘A</kbd>
+          </ListItem>
+          <ListItem
+            summaryText="Make a commit"
+            id="step-4"
+            openId={this.state.openId}
+            onClick={this.handleToggle}
+          >
+            <div>
+              Write a message that describes the changes you made. When you’re
+              done, click the commit button to finish.
+            </div>
+            <kbd>⌘ Enter</kbd>
+          </ListItem>
+          <ListItem
+            summaryText="Push to GitHub"
+            id="step-5"
+            openId={this.state.openId}
+            onClick={this.handleToggle}
+          >
+            <div>
+              Pushing your commits updates the repository on GitHub with any
+              commits made on your computer to a branch.
+            </div>
+            <kbd>⌘P</kbd>
+          </ListItem>
+          <ListItem
+            summaryText="Open a pull request"
+            id="step-6"
+            openId={this.state.openId}
+            onClick={this.handleToggle}
+          >
+            <div>
+              Pull Requests are how you propose changes. By opening one, you’re
+              requesting that someone review and merge them.
+            </div>
+            <Button onClick={this.openPullRequest}>Open pull request</Button>
```

Comment:
This should be title case on macOS and sentence case on Windows.
```suggestion
            <Button onClick={this.openPullRequest}>
              {__DARWIN__ ? 'Open Pull Request' : 'Open pull request'}
            </Button>
```